NF.jsonld

```diff
@@ -3549,26 +3552,50 @@
                 {
                     "@id": "bts:LowGradeGlioma"
                 },
+                {
+                    "@id": "bts:NOS"
+                },
+                {
+                    "@id": "bts:PilocyticAstrocytoma"
+                },
+                {
+                    "@id": "bts:PilomyxoidAstrocytoma"
+                },
                 {
                     "@id": "bts:DiffuseAstrocytoma"
                 },
                 {
-                    "@id": "bts:AnaplasticAstrocytoma"
+                    "@id": "bts:GlioblastomaMultiforme"
                 },
                 {
-                    "@id": "bts:Glioblastoma"
+                    "@id": "bts:AnaplasticPilocyticAstrocytoma"
                 },
                 {
-                    "@id": "bts:Schwannoma"
+                    "@id": "bts:PleomorphicXanthoastrocytoma"
                 },
                 {
-                    "@id": "bts:Meningioma"
+                    "@id": "bts:Oligoastrocytoma"
                 },
                 {
-                    "@id": "bts:Normal"
+                    "@id": "bts:High-GradeGlioma"
                 },
                 {
-                    "@id": "bts:Teratoma"
+                    "@id": "bts:NOS"
```

Comment:
Quoting doesn't work

Reply:
Interesting! I could have sworn that works, but obviously mistaken. Thanks for checking!